x-pack/plugins/canvas/public/components/color_dot/color_dot.js

```diff
@@ -11,7 +11,14 @@ export const ColorDot = ({ value, children }) => {
   return (
     <div className="canvasColorDot">
       <div className="canvasColorDot__background canvasCheckered" />
-      <div className="canvasColorDot__foreground" style={{ background: value }}>
+      <div
+        className={
+          value === ''
+            ? 'canvasColorDot__foreground canvasColorEmpty'
```

Comment:
Probably don't need this conditional anymore since the background color default is back to white.